packages/engine-core/src/data-proxy/utils/getClientVersion.ts

```diff
@@ -13,5 +13,5 @@ export function getClientVersion(config: EngineConfig) {
     return version
   }
 
-  return '3.2.0' // and we default it to this one if does not
+  return '3.4.0' // and we default it to this one if does not
```

Comment:
Bump to latest client.

Reply:
Update with `3.4.x` branch and then bump this value to `3.4.2` (today's patch release)